Scripts/RTS/World.cs

```diff
@@ -5,50 +5,50 @@ namespace RTS;
 
 public partial class World : Node
 {
-    public static World Instance { get; private set; }
```

Comment:
Please revert these indentation changes. I prefer 4 spaces over tab.

Reply:
These indentation changes also make it extremely difficult to see what kind of changes you made to World.cs